runtime/oti/VMHelpers.hpp

```diff
@@ -1450,6 +1458,41 @@ class VM_VMHelpers
 	{
 		return (J9SFJNINativeMethodFrame*)((UDATA)currentThread->sp + (UDATA)currentThread->literals);
 	}
+
+	/**
+	 * Checks whether a ROM method is <clinit> or <init>
+	 *
+	 * @param romMethod[in] the J9ROMMethod to test
+	 * @param isStatic[in] true to check for <clinit>, false to check for <init>
+	 *
+	 * @returns true if the method is a constructor, false if not
+	 */
+	static VMINLINE bool
+	romMethodIsConstructor(J9ROMMethod *romMethod, bool isStatic)
```

Comment:
Constructor doesn't seem like the right name here - Initializer maybe?

Reply:
Updated.